index.html

```diff
@@ -659,6 +659,18 @@ <h2>Version 2</h2>
       <section>
         <h2>Credential Metadata</h2>
         <p><a data-cite="rfc7519#section-5.1">typ</a> MUST use the content type <code>verifiable-credential+jwt</code>.</p>
+        <p>
+          The content type <code>vc+jwt</code> implies the use of an `@context` that defines the registered claims present 
+          in a JWT to provide semantic interoperabilite when using a `vc-jwt` in an RDF capacity.  
```

Comment:
```suggestion
          in a JWT to provide semantic interoperability when using a `vc-jwt` in an RDF capacity.  
```

Or possibly "a `vc-jwt` that can be deterministically mapped to RDF" ?

Reply:
i like that deterministic langugage

Reply:
@bumblefudge would `in a JWT to provide semantic interoperability when using a `vc-jwt` and to provide a deterministic method to convert a JWT into RDF using the base [[vc-data-model]] context.` work?

Reply:
```suggestion
          in a JWT that provides semantic interoperability and deterministic conversion to RDF via the [[vc-data-model]] and its base context.```
          😎